Sources_v3/StreamChatUI/ChatChannel/MessageComposerSuggestionsCollectionViewLayout.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class MessageComposerSuggestionsCollectionViewLayout: UICollectionViewFlowLayout {
+    override public required init() {
+        super.init()
+        commonInit()
+    }
+
+    public required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        commonInit()
+    }
+
+    private func commonInit() {
+        minimumInteritemSpacing = 0
+        minimumLineSpacing = 0
+    }
+
+    override open func prepare() {
+        super.prepare()
+
+        estimatedItemSize = .init(
+            width: collectionView?.bounds.width ?? 0,
+            height: 1000
```

Comment:
This is estimated "item" size, so rather than 1000 it should be something like 44 or 50.